mab2rec/rec.py

```diff
@@ -288,7 +288,7 @@ def predict_expectations(self, contexts: Union[None, List[List[Num]], np.ndarray
         return self.mab.predict_expectations(contexts)
 
     def recommend(self, contexts: Union[None, List[List[Num]], np.ndarray, pd.Series, pd.DataFrame] = None,
-                  excluded_arms: List[List[Arm]] = None, return_scores: bool = False) \
+                  excluded_arms: List[List[Arm]] = None, return_scores: bool = False, transform_scores: bool = True) \
```

Comment:
can you remind me again why this was useful to have this option? 

re: naming, given that this is a boolean.. is_sigmoid? has_sigmoid? has_sigmoid_transform? is_sigmoid_applied?

Reply:
is this a "breaking" change now, OR we are okay since it has a default setup as True

Reply:
No, not breaking. Optional argument and default behavior is the unchanged.

Reply:
re: naming - how about "apply_sigmoid"?

Reply:
@wddcheng when you get a chance, can you change this to apply_sigmoid, merge the PR, and close the related Issues please?

Reply:
will do!